src/widgets/splits/Split.cpp

```diff
@@ -669,24 +669,18 @@ void Split::addShortcuts()
                  {
                      mode = 1;
                  }
-                 else
-                 {
-                     mode = 2;
-                 }
              }
 
-             if (mode == 0)
-             {
-                 this->setModerationMode(false);
-             }
-             else if (mode == 1)
-             {
-                 this->setModerationMode(true);
-             }
-             else
+             switch (mode)
              {
-                 this->setModerationMode(!this->getModerationMode());
+                 case 0:
+                 case 1:
+                     this->setModerationMode((bool)mode);
+                     break;
```

Comment:
I'd rather case 0 and case 1 are separate calls to `this->setModerationMode`

The rest of this file looks good